### ISSUE TITLE
[ENHANCE] Mekanism Gas

### ISSUE THREAD

**Issue by cloakable** [DarkholmeTenk/TardisModIssues#321]
Being able to pipe mekanism gas into a roundlet without having to convert it first would be really useful.

(I'm thinking fusion reactor in a separate room to the deuterium/tritium generation, also piping in oxygen/hydrogen chloride/sulfuric acid to an ore processing room)

**Comment by AlienXtream** [comment]
arnt the "gasses" technically fluids? if so the yshould work with the tank roundel

**Comment by cloakable** [comment]
Nope, they won't go into tanks, and require their own transport (mekanism pressurized tubes and ender io gas conduit).

**Comment by AlienXtream** [comment]
well then.

**Comment by SomeBadGamer** [comment]
It would make sense  and just a idea if the tardis  has the gas in its internal  storage  and it received  a certain  amount  of  damage  some of it could either disappear like it leaked  out or put some of it in the console room like a fuel pipe broke and it  let the gad out also their be a "vent" system  that can absorb  the gas kind of like  in the episode  let's kill hittler

**Comment by AlienXtream** [comment]
probably too in depth for most people.

**Comment by FoxPotato** [comment]
the only reason there was gas in the episode let's kill hitler was bc melody shot the console. And I'd like to think timelords might be smart enough to manufacture tanks which can endure some hits :p

**Comment by SomeBadGamer** [comment]
I mean a lot not you miss one button and everything  goes wrong
